rocAL/rocAL/include/augmentations/color_augmentations/node_saturation.h

```diff
@@ -25,18 +25,16 @@ THE SOFTWARE.
 #include "parameter_factory.h"
 #include "parameter_vx.h"
 
-
-class SatNode : public Node
-{
+class SaturationNode : public Node {
 public:
-    SatNode(const std::vector<Tensor *> &inputs, const std::vector<Tensor *> &outputs);
-    SatNode() = delete;
+    SaturationNode(const std::vector<Tensor *> &inputs, const std::vector<Tensor *> &outputs);
+    SaturationNode() = delete;
     void init(float sat);
     void init(FloatParam *sat);
```

Comment:
Please rename sat to sat_param if possible
Applicable to all such instances across all files